projects/counters/src/main.c

```diff
@@ -0,0 +1,38 @@
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+#define TIMER_INTERVAL_MS 500
+
+typedef struct Counters {
+  uint8_t counter_a;
+  uint8_t counter_b;
+} Counters;
+
+static void prv_counter_callback(SoftTimerId soft_timer_id, void *context) {
+  Counters *counter_ptr = context;
+
+  if (counter_ptr->counter_a % 2 == 0 && counter_ptr->counter_a != 0) {
```

Comment:
We don't need to check `counter_ptr->counter_a != 0` since we want both a and b to increment when they start.